Support YAML format for Addon advanced configuration values (#6264)
weaveworks/eksctl: userdocs/src/usage/addons.md

This returns a JSON schema of the various options available for this addon.

## Working with configuration values

> These previously discovered options can be configured by passing the `ConfigurationValues` field to the configuration file, and then used with either addon create or update commands. For the moment, all the options the user wants to configure will be passed together in the format of a JSON or YAML string, e.g.

Reviewer: ```suggestion
`ConfigurationValues` can be provided in the configuration file during the creation or update of addons. Only JSON and YAML formats are supported. 

For eg.,
```

> 

Reviewer: there's a typo in line 154 🙈

```

!!!note

>     Bear in mind that when addon configuration values are being modified, configuration conflicts will arise.

Reviewer: ⭐ ❤️
